schemas/expconf/v0/searcher.json

```diff
@@ -80,21 +84,10 @@
             ],
             "default": true
         },
-        "source_trial_id": {
-            "type": [
-                "integer",
-                "null"
-            ],
-            "default": null
-        },
-        "source_checkpoint_uuid": {
-            "type": [
-                "string",
-                "null"
-            ],
-            "default": null
-        },
```

Comment:
why relax these validations?

If you relax these validations here because it's not supported by custom searcher (why would that be the case, by the way?) then you'll need to add the validations to the non-custom searchers.